daprdocs/content/en/concepts/security-concept.md

```diff
@@ -129,5 +129,18 @@ The test focused on the following:
 
 The full report can be found [here](/docs/Dapr-july-2020-security-audit-report.pdf).
 
-Two issues, one critical and one high, were fixed during the test.
-As of July 21st 2020, Dapr has 0 criticals, 2 highs, 2 mediums, 1 low, 1 info.
+### February 2021
```

Comment:
I'd put this above the June audit (latest first)

Reply:
done